Tasks/PublishSymbols/task.json

```diff
@@ -4,6 +4,10 @@
     "friendlyName": "Index Sources & Publish Symbols",
     "description": "Index your source code and publish symbols to a file share",
     "category": "Build",
+    "visibility": [
+                    "Preview",
```

Comment:
PublishSymbols should be marked for Build only (not Preview).  It should have the same visibility as VSBuild.